extensions/ads-windows-support/README.md

```diff
@@ -0,0 +1,17 @@
+# ADS Windows Support
```

Comment:
I would update the name of the extension and the readme.  First I would remove `ads` from the extension name since all the extensions are `ads`.  Also, I would probably go with something more like "Database Admin Tools for Windows" or something like that, since "Windows Support" is a bit too broad for what the extension is doing (i.e., we have other Windows-specific functionality that isn't in the extension).

Also, we shouldn't use the acronym `ADS` in any user face UI or docs.  We should fully write out "Azure Data Studio" per marketing guidance.